### PR TITLE
Add modal window

### PR DESCRIPTION
**Pull request by Motzart**

### CODE REVIEW

**Review by IlyaVi — DISMISSED** [review]

File: app/components/common/CorneredContainer.js [hunk 1]
```diff
@@ -60,18 +60,19 @@ type Props = {
...
   headerIcon?: Object,
   subHeader?: string,
+  hasError?: boolean,
   useEmptyWrap: boolean
 };
 
 class CorneredContainer extends PureComponent<Props> {
   render() {
-    const { children, width, height, header, headerIcon, subHeader, useEmptyWrap } = this.props;
+    const { children, width, height, header, headerIcon, subHeader, hasError, useEmptyWrap } = this.props;
```
Comment (IlyaVi) [inline app/components/common/CorneredContainer.js:69]: change 'hasError' to 'isErrorModal', more meaningful name

File: app/components/common/CorneredContainer.js [hunk 2]
```diff
@@ -60,18 +60,19 @@ type Props = {
...
 
     return useEmptyWrap ? (
       <DivWrapper width={width} height={height}>
         <HeaderWrapper>
           {headerIcon && <HeaderIcon src={headerIcon} />}
-          <Header>{header}</Header>
+          <Header color={hasError ? smColors.orange : color1}>{header}</Header>
```
Comment (IlyaVi) [inline app/components/common/CorneredContainer.js:75]: typo 'color**1**'

File: app/components/common/Modal.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  hasError?: boolean,
+  children: React.node
+};
+
+class Modal extends Component<Props, State> {
+  render() {
+    const { header, subHeader, hasError, children } = this.props;
+    return (
+      <Wrapper>
+        <CorneredContainer width={520} height={310} header={header} subHeader={subHeader} hasError={hasError}>
```
Comment (IlyaVi) [inline app/components/common/Modal.js:44]: make width and height params and add default values for these

File: app/components/common/CorneredContainer.js (hunk 1 above)
Comment (Motzart) [reply app/components/common/CorneredContainer.js]: fix

File: app/components/common/Modal.js (hunk 3 above)
Comment (Motzart) [reply app/components/common/Modal.js]: made

File: app/components/common/CorneredContainer.js (hunk 2 above)
Comment (Motzart) [reply app/components/common/CorneredContainer.js]: fix

File: app/components/common/CorneredContainer.js [hunk 4]
```diff
@@ -60,18 +60,19 @@ type Props = {
...
   render() {
-    const { children, width, height, header, headerIcon, subHeader, useEmptyWrap } = this.props;
+    const { children, width, height, header, headerIcon, subHeader, isErrorModal, useEmptyWrap } = this.props;
 
     return useEmptyWrap ? (
       <DivWrapper width={width} height={height}>
         <HeaderWrapper>
           {headerIcon && <HeaderIcon src={headerIcon} />}
-          <Header>{header}</Header>
+          <Header color={isErrorModal ? smColors.orange : headerColor}>{header}</Header>
```
Comment (IlyaVi) [inline app/components/common/CorneredContainer.js:75]: static defaultProps = {
    isErrorModal: false
};
or
static defaultProps = {
    headerColor: color1
};
second is even better - more generic

Comment (Motzart) [reply app/components/common/CorneredContainer.js:80]: fixed

**Review by IlyaVi — APPROVED** [review]

**Review by IlyaVi — APPROVED** [review]

**Pull request merged**